paper-fab.html

```diff
@@ -165,7 +149,32 @@
       mini: {
         type: Boolean,
         value: false
+      },
+
+      /**
+       * If true, the button should be styled with a shadow.
+       */
+      raised: {
+        type: Boolean,
+        value: true,
+        readOnly: true
+      }
+    },
+
+    observers: [
+      `_focusedChanged(focused)`
+    ],
+
+    ready: function() {
+      if (!this.hasAttribute('role')) {
+        this.setAttribute('role', 'button');
       }
+      this._focusedChanged();
```

Comment:
This is to display the initial elevation, from `raised`. `paper-button`, for example has `onRaisedChanged` but I didn't think it was needed here because `raised` is read only and can't actually change...